app/models/user.rb

```diff
@@ -3,6 +3,8 @@ class User < ApplicationRecord
   has_many :comments
   has_many :likes
 
+  validates :name, presence: true
+
```

Comment:
- [ ] Kindly, you should also validate the `PostsCounter` as the requirements say that `it must be an integer greater than or equal to zero.`
Example :point_down: 

```rb
   validates_numericality_of :posts_counter, only_integer: true, greater_than_or_equal: 0
```